integrations/docker/images/chip-build-android/Dockerfile

```diff
@@ -21,6 +26,15 @@ RUN set -x \
     && test -d /opt/android/sdk/platforms/android-21 \
     && : # last line
 
+# Download and install android command line tool (for installing `sdkmanager`)
+RUN set -x \
+    && wget -O /tmp/android-tools.zip https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip \
+    && cd /opt/android/sdk \
+    && unzip /tmp/android-tools.zip \
+    && rm -f /tmp/android-tools.zip \
+    && test -d /opt/android/sdk/tools \
+    && : # last line
+
```

Comment:
We need `sdkmanager` to download Android SDKs and it comes with this Android SDK tool package.